dxr/plugins/python/tests/test_modules_packages/test_modules_packages.py

```diff
@@ -0,0 +1,49 @@
+from dxr.testing import DxrInstanceTestCase
+
```

Comment:
Oh, and I suppose we could rename the test folder from test_modules_packages to test_modules.